sale_automatic_workflow/tests/test_automatic_workflow.py

```diff
@@ -7,12 +7,10 @@
 from freezegun import freeze_time
 
 from odoo import fields
-from odoo.tests import tagged
 
 from .common import TestAutomaticWorkflowMixin, TestCommon
 
 
-@tagged("post_install", "-at_install")
```

Comment:
why removing?

Reply:
The PR description explains that this is due to interference of queue_job (and presumably this module), but @chaule97 isn't that what https://github.com/OCA/sale-workflow/blob/18.0/sale_automatic_workflow/tests/test_automatic_workflow.py#L23-L26 is for? Currently, the tests are depending on a preinstalled chart of accounts whereas they should be inheriting from AccountTestInvoicingCommon which requires to be run post-install. This can be considered technical debt, and we should not be moving further away from its implementation.